webapp/src/index.js

```diff
@@ -20,7 +23,23 @@ class Plugin {
             'Start Webex Meeting'
         );
         registry.registerPostTypeComponent('custom_webex', PostTypeWebex);
+        Client.setServerRoute(getServerRoute(store.getState()));
     }
 }
 
 window.registerPlugin(pluginId, new Plugin());
+
+const getServerRoute = (state) => {
+    const config = getConfig(state);
+
+    let basePath = '';
+    if (config && config.SiteURL) {
+        basePath = new URL(config.SiteURL).pathname;
+
+        if (basePath && basePath[basePath.length - 1] === '/') {
+            basePath = basePath.substr(0, basePath.length - 1);
+        }
+    }
+
+    return basePath;
+};
```

Comment:
Seems like an unrelated change from https://github.com/mattermost/mattermost-plugin-webex/pull/63

Reply:
looks like the commits got mixed up on this one. Will update for review